catalyst/dl/scripts/swa.py

```diff
@@ -11,10 +11,17 @@ def build_args(parser: ArgumentParser):
     """Builds the command line parameters."""
     parser.add_argument("--logdir", type=Path, help="Path to models logdir")
     parser.add_argument(
-        "--models_mask", "-m", type=str, help="Pattern for models to average"
+        "--models-mask",
+        "-m",
+        type=str,
+        default="train*",
```

Comment:
🤔  what do you think about `*.pth`? I mean, in this case we will take into account all possible checkpoints
```suggestion
        default="*.pth",
```